### PR TITLE
Integrate trace Operations into storage layer [full ci]

### PR DESCRIPTION
**Pull request by fdawg4l**
This integrates `trace.Operations` into the storage layer.  This change should be a NOOP except for some extra logging at the time of `context deadline exceeded`.

### CODE REVIEW

**Comment by caglar10ur** [comment]
LGTM

**Comment by hickeng** [comment]
@fdawg4l looks like this last build failed due to the renaming of the tether debug log.

File: lib/apiservers/portlayer/restapi/handlers/storage_handlers.go [hunk 1]
```diff
@@ -99,35 +103,36 @@ func (handler *StorageHandlersImpl) Configure(api *operations.PortLayerAPI, hand
...
+	api.StorageVolumeJoinHandler = storage.VolumeJoinHandlerFunc(h.VolumeJoin)
+	api.StorageListVolumesHandler = storage.ListVolumesHandlerFunc(h.VolumesList)
+	api.StorageGetVolumeHandler = storage.GetVolumeHandlerFunc(h.GetVolume)
 }
 
 // CreateImageStore creates a new image store
-func (handler *StorageHandlersImpl) CreateImageStore(params storage.CreateImageStoreParams) middleware.Responder {
-	url, err := storageImageLayer.CreateImageStore(context.TODO(), params.Body.Name)
+func (h *StorageHandlersImpl) CreateImageStore(params storage.CreateImageStoreParams) middleware.Responder {
+	op := trace.NewOperation(context.Background(), fmt.Sprintf("CreateImageStore(%s)", params.Body.Name))
```
Comment (hickeng) [inline lib/apiservers/portlayer/restapi/handlers/storage_handlers.go]: If params has a context, can we use that to chain off? That way we'd get cross layer stacks.
I'm not sure what that'd look like, but could be useful. Could also be excessive - probably not for this PR either way.

**Comment by hickeng** [comment]
LGTM

**Comment by fdawg4l** [comment]
.

**Comment by fdawg4l** [comment]
lgtm

**Pull request merged**